### ISSUE TITLE
Cannot view videos online inside shared links

### ISSUE THREAD

**Issue by t123yh** [owncloud/core#25795]
### Steps to reproduce
1. Put a MP4 on your ownCloud inside a folder
2. Share that folder with link
3. Try to watch that video online

### Expected behaviour
We should be able to watch the video.

### Actual behaviour
We can't watch the video(server returns an 500)

### Server configuration
**Operating system**: Ubuntu 16.04
**Web server:** Apache
**Database:** MySQL
**PHP version:** php 7.0
**ownCloud version:** (see ownCloud admin page) ownCloud 9.1.0 (stable)
**Updated from an older ownCloud or fresh install:** updated from 9.x (I can't remember)
**Where did you install ownCloud from:** apt repo

#### ownCloud log (data/owncloud.log)
```
{"reqId":"qoAGc003uxQhS9qPq\/vG","remoteAddr":"x.x.x.x","app":"index","message":"Exception: {\"Exception\":\"Error\",\"Message\":\"Call to undefined function split()\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/legacy\\\/files.php(256): OC_Files::parseHttpRangeHeader('318552-14006664', 14171079)\\n#1 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/legacy\\\/files.php(119): OC_Files::getSingleFile(Object(OC\\\\Files\\\\View), '\\\/\\\\xE5\\\\x85\\\\xB1\\\\xE4\\\\xBA\\\\xAB\\\/\\\\xE9\\\\x95\\\\xBF\\\\xE8\\\\x80\\\\x85\\\\xE5...', '\\\\xE8\\\\x9B\\\\xA4\\\\xE8\\\\x9B\\\\xA4\\\\xE9\\\\x99\\\\x84\\\\xE8\\\\xBA\\\\xAB\\\\xE8\\\\xAF\\\\xB8...', Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_sharing\\\/lib\\\/Controllers\\\/ShareController.php(503): OC_Files::get('\\\/\\\\xE5\\\\x85\\\\xB1\\\\xE4\\\\xBA\\\\xAB\\\/\\\\xE9\\\\x95\\\\xBF\\\\xE8\\\\x80\\\\x85\\\\xE5...', '\\\\xE8\\\\x9B\\\\xA4\\\\xE8\\\\x9B\\\\xA4\\\\xE9\\\\x99\\\\x84\\\\xE8\\\\xBA\\\\xAB\\\\xE8\\\\xAF\\\\xB8...', Array)\\n#3 [internal function]: OCA\\\\Files_Sharing\\\\Controllers\\\\ShareController->downloadShare('ZLeB1EXLujKOOPK', '\\\\xE8\\\\x9B\\\\xA4\\\\xE8\\\\x9B\\\\xA4\\\\xE9\\\\x99\\\\x84\\\\xE8\\\\xBA\\\\xAB\\\\xE8\\\\xAF\\\\xB8...', '\\\/', '')\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/AppFramework\\\/Http\\\/Dispatcher.php(159): call_user_func_array(Array, Array)\\n#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/AppFramework\\\/Http\\\/Dispatcher.php(89): OC\\\\AppFramework\\\\Http\\\\Dispatcher->executeController(Object(OCA\\\\Files_Sharing\\\\Controllers\\\\ShareController), 'downloadShare')\\n#6 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/AppFramework\\\/App.php(110): OC\\\\AppFramework\\\\Http\\\\Dispatcher->dispatch(Object(OCA\\\\Files_Sharing\\\\Controllers\\\\ShareController), 'downloadShare')\\n#7 \\\/var\\\/www\\\/owncloud\\\/lib\\\/public\\\/AppFramework\\\/App.php(135): OC\\\\AppFramework\\\\App::main('ShareController', 'downloadShare', Object(OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer))\\n#8 \\\/var\\\/www\\\/owncloud\\\/core\\\/routes.php(140): OCP\\\\AppFramework\\\\App->dispatch('ShareController', 'downloadShare')\\n#9 [internal function]: OC\\\\Route\\\\Router->{closure}(Array)\\n#10 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/Route\\\/Router.php(280): call_user_func(Object(Closure), Array)\\n#11 \\\/var\\\/www\\\/owncloud\\\/lib\\\/base.php(891): OC\\\\Route\\\\Router->match('\\\/s\\\/ZLeB1EXLujKO...')\\n#12 \\\/var\\\/www\\\/owncloud\\\/index.php(39): OC::handleRequest()\\n#13 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/legacy\\\/files.php\",\"Line\":195}","level":3,"time":"2016-08-14T12:36:13+00:00","method":"GET","url":"\/index.php\/s\/ZLeB1EXLujKOOPK\/download?path=%2F&files=%E8%9B%A4%E8%9B%A4%E9%99%84%E8%BA%AB%E8%AF%B8%E8%91%9B%E4%BA%AE%E9%AA%82%E6%AD%BB%E7%8E%8B%E6%9C%97.mp4","user":"--"}
```

**Comment by RealRancor** [comment]
@t123yh Duplicate of https://github.com/owncloud/core/issues/25780 and https://github.com/owncloud/core/issues/25483 which gets fixed with the upcoming 9.1.1

**Comment by t123yh** [comment]
That's great. Sorry to disturb.
